src/core/core_utils.js

```diff
@@ -670,6 +670,50 @@ function getSizeInBytes(x) {
   return Math.ceil(Math.ceil(Math.log2(1 + x)) / 8);
 }
 
+function getVerticalPresentationForm() {
```

Comment:
Can we please place this new functionality in https://github.com/mozilla/pdf.js/blob/master/src/core/fonts_utils.js instead?

Reply:
Also, the way that this uses the `getLookupTableFactory` is slightly inconsistent (and more complicated) compared to all existing cases.
Can this please be updated to use the same pattern as in e.g. https://github.com/mozilla/pdf.js/blob/ccad2f889ae8e721b1a1119190f03f99c68ae763/src/core/standard_fonts.js#L101-L120 instead, since that should mean less parsing as well?

Reply:
I've just tested locally and as written this code causes the lookup-table to be recomputed on *every invocation* of `getVerticalPresentationForm`, hence this *must* be re-written as outlined above.